webapp/actions/user_actions.jsx

```diff
@@ -277,3 +277,20 @@ export function searchUsers(term, teamId = TeamStore.getCurrentId(), options = {
         }
     );
 }
+
+export function autocompleteUsers(username, channelId = '', success, error) {
```

Comment:
Does this need the additional error handling that all the calls in `AsyncClient` have? Also, I don't think having a default parameter works if it's not the last parameter